sdk/tables/Azure.Data.Tables/src/Sas/TableSasBuilder.cs

```diff
@@ -47,20 +47,13 @@ public TableSasBuilder(string tableName, string rawPermissions, DateTimeOffset e
         }
 
         /// <summary>
-        /// Initializes an instance of a <see cref="TableSasBuilder"/>.
+        /// Initializes an instance of a <see cref="TableSasBuilder"/> with no details set. At minimum, additional properties must be set to create a valid SaS.
```

Comment:
do we throw a clear exception in ToSasQueryParameters if the object is not properly initialized?

Reply:
Yes - we validate and throw [here](https://github.com/christothes/azure-sdk-for-net/blob/06ae64e74747ccb9ff5a694a8bb47a0f5533dda1/sdk/tables/Azure.Data.Tables/src/Sas/TableSasBuilder.cs#L300)